pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go

```diff
@@ -26,3 +26,15 @@ func (cm *ClusterManager) AddAttachbleHost(hostname string) error {
 
 	return nil
 }
+
+func (cm *ClusterManager) GetNextAttachableHost(currentHost string) string {
+	if len(cm.AttachableHosts) == 0 {
+		return ""
```

Comment:
please rename the tile to nvmeof-recoverer: reassign faulted OSD to the attachable host